src/components/table/table.types.ts

```diff
@@ -28,6 +29,11 @@ export interface Column<T extends object = any> {
      * A component used to render inside the column header
      */
     headerComponent?: TableComponentDefinition;
+
+    /**
+     * Sets the horizontal text alignment for the column (left|center|right)
+     */
+    hozAlign?: Tabulator.ColumnDefinitionAlign;
```

Comment:
Kind of nitpicky, but I don't really like the shortening `hozAlign`. (I understand it comes from Tabulator.) I think I'd like it to be either `horizontalAlign` or, if we want to go short, just `hAlign`… 🤔

@jgroth Thoughts?

Reply:
I agree with you. I was about to write it like that at the beginning. When I updated the examples I had to use `hozAlign` . For example `{ title: 'Height', field: 'height', hozAlign: 'right' },`. I thought of consistency that is the reason I used same name everywhere

Reply:
Hmm… ok. I'll have a quick look at it, and if I can't figure out how to make it work, we'll just go with `hozAlign` 🙂

Reply:
Yeah I don't like `hozAlign` either. I cannot see why we would not be able to use `horizontalAlign` :thinking:

Reply:
Also, we should not expose types from Tabulator like this. If they make changes to it we might make breaking changes in our own API without realizing it.

Reply:
I pushed fixups for making it `hAlign`, but it's easy to go from there to `horizontalAlign`.

Reply:
I changed it to `horizontalAlign`.

Reply:
> Also, we should not expose types from Tabulator like this. If they make changes to it we might make breaking changes in our own API without realizing it.

What do you think if we have our columnDefinationAlign then?
Like this `type ColumnDefinitionAlign = 'left' | 'center' | 'right';`

Reply:
I think we can just inline it for now. Or do you think it's useful to have a separate type for it?

Reply:
Yeah, I tried with this:

```ts
    /**
     * Sets the horizontal text alignment for the column
     */
    horizontalAlign?: 'left' | 'center' | 'right';
```

and it looks like this:

![image](https://user-images.githubusercontent.com/154725/131465944-1641bfb9-353a-4596-af58-c1389fab20b8.png)

👌

Reply:
We can squash those changes and merge.